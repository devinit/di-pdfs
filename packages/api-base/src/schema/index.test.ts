import * as prettyFormat from 'pretty-format';
import {getResolvers} from '.';

describe('Schema', () => {
    it('getting resolvers', async () => {
       const data = await getResolvers('src/schema/**/resolver.js');
       expect(prettyFormat(data)).toMatchSnapshot();
    }, 1000);
});
