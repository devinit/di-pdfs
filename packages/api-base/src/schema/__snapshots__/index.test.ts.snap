// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schema getting resolvers 1`] = `
"Object {
  \\"Query\\": Object {
    \\"dvptCooperation\\": [Function dvptCooperation],
    \\"tchCooperation\\": [Function tchCooperation],
  },
}"
`;
